PaWPal/PaWPal/RegisterViewController.swift

```diff
@@ -25,18 +25,22 @@ class RegisterViewController: UIViewController {
         // Dispose of any resources that can be recreated.
     }
     
+    // Registers new users - unless fields are invalid
+    // Currently, a single device's login data is stored using NSUserDefaults
+    // Eventually, login data for multiple devices/users will be stored in a database
     @IBAction func createRegistration(sender: AnyObject) {
         
-        // Retrieve user's email and passwords
         let userEmail = userEmailTextField.text
         let userPassword = userPasswordTextField.text
         let userRepeatPassword = userRepeatPasswordTextField.text
         
         // Check validity
-        // TODO: check valid email address
         if (userEmail!.isEmpty || userPassword!.isEmpty || userRepeatPassword!.isEmpty) {
             displayAlert("All fields must be filled", handler: nil)
         }
+        if (!userEmail!.isValidEmail()) {
```

Comment:
i think that's just a swift thing. text fields are just optional values and we have to unwrap it safely. still trying to fiddle with things